src/app.ts

```diff
@@ -2,6 +2,9 @@ import * as Koa from 'koa';
 import * as bodyParser from 'koa-bodyparser';
 import * as helmet from 'koa-helmet';
 import * as Debug from 'debug';
+import fs = require('fs');
```

Comment:
Can we use es6 module import syntax
```suggestion
import * as fs from 'fs';
```